app/dashboard/views.py

```diff
@@ -83,14 +83,24 @@ def new_interest(request, bounty_id):
     profile_id = request.session.get('profile_id')
     if not profile_id:
         return JsonResponse(
-            {'error': 'You must be authenticated!'},
+            {'error': 'You must be authenticated via github to use this feature!'},
             status=401)
 
     try:
         bounty = Bounty.objects.get(pk=bounty_id)
     except Bounty.DoesNotExist:
         raise Http404
 
+    num_issues = 3
+    active_bounties = Bounty.objects.filter(idx_status__in=['open', 'started'])
+    num_active = Interest.objects.filter(profile_id=profile_id, bounty__in=active_bounties).count()
+    is_working_on_too_much_stuff = num_active > num_issues
```

Comment:
Will we want to check `>=` here?

Reply:
yes

Reply:
ffcedbfaa204deb8c256e5a8a09f4d8cbe558981